kb/2025-05-08/using-pod-security-standard.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Is this file needed if we have the other one - `kb/using-pod-security-standard.md`?

Reply:
nope. fixed it.